### ISSUE TITLE
Sort by student written

### ISSUE THREAD

**Issue by nhyty** [newtheatre/history-project#478]
A page which shows all student written pieces done here.

**Comment by wjdp** [comment]
:+1:

**Comment by wjdp** [comment]
https://history.newtheatre.org.uk/playwrights/ could have a checkbox for 'only show student written'.

**Comment by nhyty** [comment]
Yes please, would be nice to show off all the student written wroks we have put on

**Comment by nhyty** [comment]
Any progress on this one?

**Comment by wjdp** [comment]
Did some work on this a while back, will fig up and finish.

**Comment by nhyty** [comment]
Any chance of this one, would be nice for us to show off how much student writing we do :)

**Comment by wjdp** [comment]
Will have a look at at some point. Two ways of going about this:

- **Checkbox on playwrights page**. Less ui addition, doesn't showcase quite as much.
- **Separate page showing only student writing.** Another page, but highlights more.

**Comment by nhyty** [comment]
I feel like the first one is good enough.
